purchases/src/main/kotlin/com/revenuecat/purchases/models/InstallmentsInfo.kt

```diff
@@ -0,0 +1,16 @@
+package com.revenuecat.purchases.models
+
+/**
+ * Type containing information of installment subscriptions. Currently only supported in Google Play.
+ */
+interface InstallmentsInfo {
+    /**
+     * Number of payments the customer commits to in order to purchase the subscription.
+     */
+    val commitmentPaymentsCount: Int
+
+    /**
+     * After the commitment payments are made, the number of payments the user commits to upon a renewal.
+     */
+    val subsequentCommitmentPaymentsCount: Int
```

Comment:
Hmm, not sure... While the type would be an indication, if we did `commitmentPayments`, there is nothing in the name indicating that this is number of payments/periods/... IMO it might be slightly more confusing, and I would more likely need to checkout the docs... Wdyt? Happy to chat more about it!

Reply:
No you're right! Concise is good, but not at the cost of clarity imo 🙂